packages/server/modules/fileuploads/services/management.ts

```diff
@@ -83,25 +50,10 @@ export const insertNewUploadAndNotifyFactoryV2 =
     queues: Pick<FileImportQueue, 'scheduleJob' | 'supportedFileTypes'>[]
     pushJobToFileImporter: PushJobToFileImporter
     saveUploadFile: SaveUploadFileV2
-    publish: PublishSubscription
     emit: EventBusEmit
   }): InsertNewUploadAndNotifyV2 =>
   async (upload) => {
     const file = await deps.saveUploadFile(upload)
-
```

Comment:
the bug was here - v2 version of function did not emit all of the necessary GQL subs